### PR TITLE
feat(theme): added the custom theme functionality in the navbar

### PR DESCRIPTION
**Pull request by Shruti3004**
## Description

Added the team toggle functionality in the navbar.

### Changes

- Created the context for the global state maintenance.
- Added the light and dark theme option in Header.

## How to test
Check the header and toggle between the themes.

## Screenshots

Screenshot from 2021-06-26 07-52-06
Screenshot from 2021-06-26 07-52-12
Screenshot from 2021-06-26 07-52-16

### CODE REVIEW

File: src/components/Header/index.js [hunk 1]
```diff
@@ -158,10 +161,24 @@ const Header = () => {
             {isAuth() ? (
               <Dropdown.Menu>
                 <Dropdown.Item onClick={handleLogout}>Log out</Dropdown.Item>
+                <Dropdown.Divider />
+                <Dropdown.Item onClick={() => setTheme("light")}>
+                  Light Theme
+                </Dropdown.Item>
+                <Dropdown.Item onClick={() => setTheme("dark")}>
+                  Dark Theme
+                </Dropdown.Item>
```
Comment (GMishx) [inline src/components/Header/index.js:170]: Do you thing removing theme based on context will be good here?
For example if I choose light theme, I can be given option of dark theme only?

**Review by GMishx — APPROVED** [review]
Changes looks good. Tested, working as expected.

File: src/components/Header/index.js (hunk 1 above)
Comment (Shruti3004) [reply src/components/Header/index.js:170]: Yeah sure we can do that

Comment (GMishx) [reply src/components/Header/index.js:170]: I am asking it as a question. What do you think about it?

Comment (Shruti3004) [reply src/components/Header/index.js:170]: Yeah that would be nice but right now we have setup only light theme dark theme, but we even had a discussion that their may be a need to add more theme variables in that case I think so we can keep it like this only.

Comment (GMishx) [reply src/components/Header/index.js:233]: Thank you for clarification. Will merge it then.

**Review by GMishx — APPROVED** [review]

**Pull request merged**